Libraries/D3D12RaytracingFallback/src/UberShaderRayTracingProgram.cpp

```diff
@@ -196,6 +203,46 @@ namespace FallbackLayer
             L"RaytracingDevice::CreateRootSignature");
     }
 
+    void UberShaderRaytracingProgram::DumpReproInformation(UINT maxAttributeSizeInBytes, std::vector<DxilLibraryInfo> librariesInfo, std::vector<LPCWSTR> exportNames)
+    {
+        std::ofstream cmdFile("fallbackcompilerrepro" + std::to_string(g_ReproNumber++) + ".cmd");
```

Comment:
Is there a particular reason you're using Ascii for this and not wide-char, since the file already works with wide-chars?

Reply:
I don't seem to be able to mix ascii and wide chars in the file, so I decided to convert the export names to ascii instead of everything else to wide char...